api/test/wfl/tools/workloads.clj

```diff
@@ -25,7 +25,8 @@
 
 (def watchers
   [["email" "hornet@broadinstitute.org"]
-   ["slack" "C026PTM4XPA"]])
+   ["slack" "C026PTM4XPA"]  ;; Private channel #hornet-slack-app-testing
+   ])
```

Comment:
Comments like this are evidence of poor design.
(Looks like the channel name should be in the data.)

Reply:
Good point.  We went with channel IDs because they are immutable, the channel names can change.  But it's a PITA to track back to the name, don't think you can do it within the Slack App at least.

Possible improvements --

- Allow a Slack channel name to be specified instead of channel ID at workload creation, internally at that point we map it to its channel ID.  [No Slack API](https://stackoverflow.com/questions/50106263/slack-api-to-find-existing-channel) for fetching a channel by name, so we'd have to fetch all channels and filter ourself.
- Allow a Slack channel name to be specified as decoration in addition to channel ID, we don't do anything with it programmatically or guarantee that it won't change in the future but helps point to where the notifications are going.

Reply:
Yes. I meant to advocate your "decoration" idea.

Reply:
The slack channel ID is just that: an identifier. We should also support a name to support debugging.

Reply:
Sweet.  I think the decorator idea may work out of the box.  I will test it out.

Reply:
> Sweet. I think the decorator idea may work out of the box. I will test it out.

The name can be optional, or default to the `::creator`, or …

Reply:
Optional name works fine, this notification was triggered in system tests with the updated `[slack <channel-id> <channel-name>]` watcher.

https://broadinstitute.slack.com/archives/C026PTM4XPA/p1643055770000300

The channel name also appears when fetching the workload:

<img width="1512" alt="Screen Shot 2022-01-24 at 3 06 02 PM" src="https://user-images.githubusercontent.com/79769153/150859368-aa4844c1-1e25-4b8c-9f54-30d216876ff0.png">

I supplemented public-facing documentation to suggest this.